whiskey.yml

```diff
@@ -3,7 +3,10 @@ Build:
 - Version:
     Path: Carbon.IIS\Carbon.IIS.psd1
     Prerelease:
-    - "*": rc1
+    - "prerelease/*": rc1
+    - "feature/*": alpha1
+    - "bugfix/*": beta1

```

Comment:
bugfix feels odd as beta given others. Can't quite nail it down.

Reply:
Bugfixes usually happen off already-released code, so I thought beta was the best modifier.